docs/ECCTOOL.md

```diff
@@ -16,11 +20,262 @@ The following subcommands are available:
+
+### schedules
+
+`ecctool schedules` subcommand is used to show the status of schedules.
+This subcommand has no mandatory parameters.
+
+#### Example
+
+In this example we will use `ecctool schedules` to check the status of schedules.
+The output shows all schedules the local ecChronos instance has.
+
+```bash
+Snapshot as of 2022-09-22 14:05:12
+-----------------------------------------------------------------------------------------------------------------------------------------------------------
+| Id                                   | Keyspace              | Table              | Status    | Repaired(%) | Completed at        | Next repair         |
+-----------------------------------------------------------------------------------------------------------------------------------------------------------
+| f7bf2960-3a6d-11ed-b3c3-4d376b8456cd | test                  | table1             | COMPLETED | 100.00      | 2022-09-16 14:05:10 | 2022-09-23 14:05:10 |
+| f8ec37b0-3a6d-11ed-b3c3-4d376b8456cd | test                  | table2             | COMPLETED | 100.00      | 2022-09-16 14:05:10 | 2022-09-23 14:05:10 |
+| fbe5efb0-3a6d-11ed-b3c3-4d376b8456cd | keyspaceWithCamelCase | tableWithCamelCase | COMPLETED | 100.00      | 2022-09-16 14:05:10 | 2022-09-23 14:05:10 |
+| fa060c20-3a6d-11ed-b3c3-4d376b8456cd | test2                 | table1             | COMPLETED | 100.00      | 2022-09-16 14:05:10 | 2022-09-23 14:05:10 |
+| fab2edf0-3a6d-11ed-b3c3-4d376b8456cd | test2                 | table2             | COMPLETED | 100.00      | 2022-09-16 14:05:10 | 2022-09-23 14:05:10 |
+-----------------------------------------------------------------------------------------------------------------------------------------------------------
+Summary: 5 completed, 0 on time, 0 blocked, 0 late, 0 overdue
+```
+
+Looking at the example output above, the columns are:
+
+`Id` - the schedule ID, this corresponds to the table id.
+
+`Keyspace` - the keyspace the repair is run on.
+
+`Table` - the table the repair is run on.
+
+`Status` - the status of the repair.
+The possible statuses are:
+* `ON_TIME` - the schedule is awaiting execution or is currently running
+* `LATE` - the schedule is late, warning time specified in the configuration has passed.
+* `OVERDUE` - the schedule is overdue, error time specified in the configuration has passed.
+* `COMPLETED` - the schedule is completed
+
+`Repaired(%)` - the number of ranges repaired within the interval vs total ranges.
+This value can go up and down as ranges become unrepaired.
+
+`Completed at` - the time when the all ranges for the schedule are repaired.
+ecChronos assumes all ranges are repaired if there's no repair history.
+
+`Next repair` - the time when the schedule will be made ready for execution.
```

Comment:
This probably requires a bit more explanation. From this it is unclear when the time is updated.